src/launcher/features/apps/appsSlice.ts

```diff
@@ -184,10 +184,10 @@ const slice = createSlice({
             state.isDownloadingLatestAppInfo = false;
         },
 
-        // Update progress
-        updateAppProgress(
+        // Update app installprogress
```

Comment:
I get the rename, but I feel that this looks like "wasted effort"
I'd just remove the comment at this point, it has the same text as the name of the function it's mean to describe